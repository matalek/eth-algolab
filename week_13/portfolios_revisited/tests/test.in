20 2
22 2
26 2
31 2
23 2
9 2
36 2
28 2
35 2
48 1
42 1
40 2
47 2
24 2
31 2
45 1
29 1
27 2
38 1
27 1
46 2
112 -16 15 15 14 5 0 8 4 -6 3 -21 21 1 -13 6 -18 -13 -6 21
-16 52 -9 -15 4 -5 0 -6 10 11 11 0 1 2 -9 12 1 13 -13 0
15 -9 95 -15 -1 5 4 17 10 -2 -3 1 -12 -21 9 -21 8 -5 15 9
15 -15 -15 98 1 16 -3 -23 5 -14 15 -12 15 0 12 -18 14 12 -1 4
14 4 -1 1 41 10 15 -17 -2 -14 10 16 -5 0 -1 -7 -10 -2 -9 21
5 -5 5 16 10 79 3 -3 -22 8 -20 -3 13 19 1 3 -18 -14 14 7
0 0 4 -3 15 3 97 -3 -16 -12 -11 4 9 7 -8 -25 3 -18 -4 4
8 -6 17 -23 -17 -3 -3 97 -3 8 -9 -13 -19 -3 2 -1 2 -4 -8 -10
4 10 10 5 -2 -22 -16 -3 94 -16 8 15 -19 3 -13 -18 6 15 -2 -5
-6 11 -2 -14 -14 8 -12 8 -16 117 1 -19 14 3 -14 27 -2 1 21 -5
3 11 -3 15 10 -20 -11 -9 8 1 135 0 4 11 14 11 -1 -7 19 11
-21 0 1 -12 16 -3 4 -13 15 -19 0 96 8 -3 -7 4 8 -7 0 15
21 1 -12 15 -5 13 9 -19 -19 14 4 8 96 7 -18 22 -3 -13 10 12
1 2 -21 0 0 19 7 -3 3 3 11 -3 7 79 -9 17 -17 -15 2 5
-13 -9 9 12 -1 1 -8 2 -13 -14 14 -7 -18 -9 32 4 -3 -7 1 -10
6 12 -21 -18 -7 3 -25 -1 -18 27 11 4 22 17 4 140 -8 -19 7 2
-18 1 8 14 -10 -18 3 2 6 -2 -1 8 -3 -17 -3 -8 50 9 -11 -2
-13 13 -5 12 -2 -14 -18 -4 15 1 -7 -7 -13 -15 -7 -19 9 65 -3 5
-6 -13 15 -1 -9 14 -4 -8 -2 21 19 0 10 2 1 7 -11 -3 79 11
21 0 9 4 21 7 4 -10 -5 -5 11 15 12 5 -10 2 -2 5 11 52
103 225
104 81

9 2
13 0
67 2
65 11
55 18
80 20
114 11
99 19
76 16
93 17
51 0 22 6 38 -43 15 -6 15
0 34 9 27 -6 -1 -1 8 10
22 9 121 50 21 -10 -3 -8 5
6 27 50 150 33 30 0 -7 -4
38 -6 21 33 125 -3 6 24 -29
-43 -1 -10 30 -3 161 -3 -23 -9
15 -1 -3 0 6 -3 55 15 -7
-6 8 -8 -7 24 -23 15 55 -15
15 10 5 -4 -29 -9 -7 -15 33
700 324
9900 739600

9 2
6 4
73 4
30 17
102 10
81 13
111 13
92 20
72 17
93 16
47 24 9 31 -8 -9 16 -23 -16
24 40 24 13 -31 -5 2 -17 26
9 24 55 -26 -28 -12 0 12 20
31 13 -26 82 -6 -43 -24 -20 -17
-8 -31 -28 -6 130 77 -40 -24 -10
-9 -5 -12 -43 77 128 -23 -21 4
16 2 0 -24 -40 -23 108 2 -9
-23 -17 12 -20 -24 -21 2 127 5
-16 26 20 -17 -10 4 -9 5 87
684 100
5556 2910436

12 2
41 1
46 6
27 3
64 6
49 8
30 8
55 5
70 7
63 9
40 6
66 8
56 9
168 6 -8 1 -24 -29 -3 22 -30 17 2 -5
6 168 -109 -31 -40 24 21 -10 9 3 2 -12
-8 -109 170 -34 41 -37 -44 -13 -4 -5 9 -2
1 -31 -34 71 20 78 12 10 4 -18 -1 24
-24 -40 41 20 80 79 -22 9 15 -3 -6 16
-29 24 -37 78 79 223 46 -1 4 3 4 12
-3 21 -44 12 -22 46 204 17 -24 -21 -18 -21
22 -10 -13 10 9 -1 17 190 -103 6 18 -4
-30 9 -4 4 15 4 -24 -103 125 -28 -48 -1
17 3 -5 -18 -3 3 -21 6 -28 176 60 29
2 2 9 -1 -6 4 -18 18 -48 60 93 90
-5 -12 -2 24 16 12 -21 -4 -1 29 90 177
6886 6241
4641 21904

26 1
88 19
98 19
89 20
97 20
96 19
100 17
68 20
97 18
88 16
100 16
100 16
84 18
98 13
83 13
61 13
81 20
90 12
99 10
87 20
100 18
65 14
82 20
89 19
99 14
91 19
100 20
1179 -48 135 -287 92 87 -255 -44 117 22 -249 116 53 121 -1 -225 116 31 -66 -172 10 134 -102 -68 -133 -58
-48 974 141 193 -87 -154 -223 -127 197 -214 -82 226 -97 -7 -64 114 -62 -127 98 172 192 59 -39 138 58 106
135 141 579 101 -83 10 24 174 160 -32 166 115 -7 158 28 -62 -102 -65 117 -221 -6 -14 -8 158 -198 -39
-287 193 101 1057 -79 -57 160 -27 -193 -124 159 105 207 -81 53 -22 -252 -29 113 199 100 78 -131 271 -116 -54
92 -87 -83 -79 406 -114 -95 21 102 26 -87 -37 61 88 79 -119 77 -13 93 5 39 22 55 -178 163 53
87 -154 10 -57 -114 679 73 -25 224 -47 -316 116 142 -40 -210 51 56 -189 -111 -3 -197 29 -22 -115 136 128
-255 -223 24 160 -95 73 867 -35 35 -150 101 -95 174 20 -116 -130 -76 177 -27 31 71 -54 -70 -88 -37 -190
-44 -127 174 -27 21 -25 -35 707 31 -29 -102 17 -54 143 -81 -214 22 -112 -190 -135 -238 -21 49 106 -10 -13
117 197 160 -193 102 224 35 31 1170 -282 -234 25 -246 74 54 0 99 -50 165 129 -83 -114 108 -134 117 103
22 -214 -32 -124 26 -47 -150 -29 -282 1249 116 -95 -121 -96 -157 28 -50 258 -158 217 25 56 93 -15 11 -78
-249 -82 166 159 -87 -316 101 -102 -234 116 961 -45 -129 163 -4 242 -138 71 240 -13 84 -46 -94 94 -186 137
116 226 115 105 -37 116 -95 17 25 -95 -45 1119 75 139 -187 -101 -13 -190 -138 -40 -252 67 -223 127 -1 -61
53 -97 -7 207 61 142 174 -54 -246 -121 -129 75 483 45 -118 -209 -14 -106 -46 -159 114 9 -212 76 -48 -105
121 -7 158 -81 88 -40 20 143 74 -96 163 139 45 989 -190 -78 -76 -69 -181 -72 -242 52 -187 -180 14 212
-1 -64 28 53 79 -210 -116 -81 54 -157 -4 -187 -118 -190 1080 116 62 76 131 -11 -130 -184 181 17 34 -32
-225 114 -62 -22 -119 51 -130 -214 0 28 242 -101 -209 -78 116 905 257 49 -111 54 -52 -4 -18 -76 -37 276
116 -62 -102 -252 77 56 -76 22 99 -50 -138 -13 -14 -76 62 257 583 72 -155 96 -181 -152 1 -121 33 43
31 -127 -65 -29 -13 -189 177 -112 -50 258 71 -190 -106 -69 76 49 72 962 163 -30 143 18 -17 28 63 11
-66 98 117 113 93 -111 -27 -190 165 -158 240 -138 -46 -181 131 -111 -155 163 816 -111 310 90 48 74 -30 213
-172 172 -221 199 5 -3 31 -135 129 217 -13 -40 -159 -72 -11 54 96 -30 -111 756 -62 -95 -15 90 214 -106
10 192 -6 100 39 -197 71 -238 -83 25 84 -252 114 -242 -130 -52 -181 143 310 -62 1107 90 104 -14 -6 -128
134 59 -14 78 22 29 -54 -21 -114 56 -46 67 9 52 -184 -4 -152 18 90 -95 90 339 30 -27 -71 -60
-102 -39 -8 -131 55 -22 -70 49 108 93 -94 -223 -212 -187 181 -18 1 -17 48 -15 104 30 596 -213 237 -37
-68 138 158 271 -178 -115 -88 106 -134 -15 94 127 76 -180 17 -76 -121 28 74 90 -14 -27 -213 954 -207 -90
-133 58 -198 -116 163 136 -37 -10 117 11 -186 -1 -48 14 34 -37 33 63 -30 214 -6 -71 237 -207 610 14
-58 106 -39 -54 53 128 -190 -13 103 -78 137 -61 -105 212 -32 276 43 11 213 -106 -128 -60 -37 -90 14 1062
501 25

0 0
