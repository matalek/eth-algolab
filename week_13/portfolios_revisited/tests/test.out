10
8
117
2491
76
3243
250
469
22
